packages/istanbul-reports/package.json

```diff
@@ -19,20 +19,18 @@
     "@babel/core": "^7.4.5",
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.6",
     "chai": "^4.2.0",
+    "cross-env": "^5.2.0",
```

Comment:
I'm unclear why this is needed?

Reply:
Removed, not sure why I added (its needed if you need to set an env variable across environments)